src/pages/Auth/ResetPasswordPage.jsx

```diff
@@ -1,10 +1,97 @@
+import React, {useState} from 'react'
+
+import {Link} from 'react-router-dom';
+
+import FormContainer from '../../components/FormContainer';
+
+
 function ResetPasswordPage() {
+
+    const [showPassword, setShowPassword] = useState(false);
+
+    function togglePasswordVisibilty() {
+        return (
+            setShowPassword(showPassword => !showPassword)
+        )
+    };
+
     return(
         <>
-            <div className="w-full text-center ">
-                <h1>
-                    The reset page appear here
-                </h1>
+             {/* Construction Image */}
+             <div className="w-full lg:w-3/6 px-4 hidden md:block">
+                <img src={require("../../assets/images/construction.svg").default} alt="Construction"></img>
+            </div>
+
+            {/* Reset Password Form */}
+            <div className="w-full lg:w-2/6">
+                <FormContainer>
+                    <form>
+                        <div className="form-group">
+                            <h1 className="uppercase font-bold text-xl text-center pb-3">
+                                Reset Password
+                            </h1>
+                            <p className="text-xs text-center pb-10">
+                                Please enter new password to reset your password
+                            </p>
+                        </div>
+
+                        <div className="form-group">
+                            <label className="form-label"
+                                htmlFor="old-password">
+                                Old Password
+                            </label>
+                            <input type="password"
+                                id="old-password"
+                                className="form-control"
+                                placeholder="Old password"
+                            />
+                        </div>
+
+                        <div className="form-group">
+                            <label className="form-label"
+                                htmlFor="new-password">
+                                New Password
+                            </label>
+                            <input type={showPassword ? "text" : "password"}
+                                id="new-password"
+                                className="form-control"
+                                placeholder="New password"
+                            />
+                            <i className="absolute right-1 top-9 cursor-pointer" onClick={togglePasswordVisibilty}>
```

Comment:
Use hero icons components instead of svg.
npm install @heroicons/react --save
![image](https://user-images.githubusercontent.com/17546288/118754902-836c3500-b885-11eb-92db-1a31e9601870.png)

https://github.com/tailwindlabs/heroicons